frontend/src/libs/components/icon/icon.tsx

```diff
@@ -1,10 +1,11 @@
-import { type IconName } from '#libs/types/types.js';
+import { type IconColor } from '#libs/enums/enums.js';
+import { type IconName, type ValueOf } from '#libs/types/types.js';
 
 import { iconNameToIcon } from './libs/maps/icon-name-to-icon.map.js';
 
 type Properties = {
   name: IconName;
-  color?: string;
+  color?: ValueOf<typeof IconColor> | undefined;
```

Comment:
```suggestion
  color?: ValueOf<typeof IconColor>;
```
`?` already means it can be undefined

Reply:
@fshabanov We were already talking about this [before](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/44#discussion_r1304655557). According to `"exactOptionalPropertyTypes": true` rule in `tsconfig` I can't pass undefined property, even if it has `?`. I can only pass mandatory `ValueOf<typeof IconColor>` or not call that attribute at all using your suggestion

Reply:
Yes, you're right, sorry